### DISCUSSION TITLE
Purpose of our DAO Website?

### DISCUSSION THREAD

**Discussion by mattjoyal** [cxndao/constellation#18] [General]
Hey all, I have been messing around with a basic website landing page & started thinking more about *why* do we want a website? What is the point? What are we trying to convey? Who is our audience? So here are some thoughts I had. I would love to hear more from others on what they think. This will help with selecting the right website host, copy, design, etc.

**Reasons**

1. First impression
    1. When asked “What is Constellation?” this site will provide the first answer
2. Recruitment tool - Attract the attention of our general audience
3. TS alum, staff, mentor
    1. of all backgrounds, from technical to new-to-crypto
    2. get Techstars’ attention
4. The landing pad for nonmember communication
    1. General questions
    2. Partners & startup investment opportunities
5. Spread the word - home for our purpose
    1. Display our purpose, charter, mission
6. Accomplishments
    1. Proudly display our achievements & successes as we grow
7. Repository of resources
    1. Links to social media, tools used by members
    2. “What is a DAO?” & other knowledge bits to encourage involvement

**Comment by galligan** [comment]
@mattjoyal This is awesome. If our "reasons" might be considered "goals" what do you think some "non-goals" might be?

**Reply by mattjoyal** [reply]
~Hey @galligan, yeah these points are mostly towards the goal of attracting more members through education & showing impact/progress.~

~As a non-goal & as @shivaas was saying, having this as a single source of informational truth for existing members as well is important. Being able to provide easy navigation, clarity, & transparency in what we are working on may not be a goal but a good end result for everyone. And yeah, better SEO while we are at it.~

**Comment by shivaas** [comment]
Great question @mattjoyal. I looked at https://www.orangedao.xyz/ as an input into my thoughts and here's what I came up with:

- a site becomes a repository (or a single page) that covers the gist of what the DAO is, who's involved and an index of links to learn more about the various pieces, access discords etc.
- websites are easily shareable and bookmarkable across a variety of mediums
- discoverability: people can hit the googles and find more about the DAO
